content/applications/inventory_and_mrp/manufacturing/subcontracting/dropship_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,187 @@
+delivery lead time, to generate the date on which the subcontractor must receive the required
+components, in order to manufacture the product and deliver it on time.
+
+In cases where components are dropshipped to the subcontractor, an additional *delivery lead time*
+can be set for each component, to generate the date that the component must be purchased, so that it
+can be dropshipped to the subcontractor by the date on which manufacturing must begin.
+
+.. important::
+   Like all lead times in Odoo, lead times for subcontracted products are only an estimate, and are
+   based on how long actions are *expected* to take.
+
+   Unforeseen circumstances can impact the completion of these actions, which means that lead times
+   should not be viewed as guarantees.
+
+Configuration
+=============
+
+When using the :doc:`Dropship Subcontractor on Order <subcontracting_dropship>` route, a company is
+responsible for purchasing the necessary components from a vendor and having them dropshipped
+directly to the subcontractor.
+
+This means that, in addition to the amount of time it takes the subcontractor to manufacture and
+deliver the product, it is also necessary to consider how long it takes the component vendor to
+dropship the components to the subcontractor.
+
+By assigning a product's subcontractor a *delivery lead time*, specifying a *manufacturing lead
+time* on the product's BoM, and assigning the vendor of the components an additional *delivery lead
+time*, *Dropship Subcontractor* orders for the product's components display the deadline for
+confirming a dropship order to send the components to the subcontractor.
+
+Product delivery lead time
+--------------------------
+
+To set a delivery lead time for a product's subcontractor, navigate to :guilabel:`Inventory app -->
+Products --> Products`, and select a subcontracted product.
+
+Then, select the :guilabel:`Purchase` tab on the product's page. If the subcontractor has not
+already been added as a vendor, do so now by clicking :guilabel:`Add a line`, and selecting the
+subcontractor in the :guilabel:`Vendor` column.
+
+Once the subcontractor has been added, enter the number of days it takes them to manufacture and
+deliver the product, in the :guilabel:`Delivery Lead Time` column.
+
+.. image:: dropship_subcontracting_lead_times/delivery-lead-time.png
+   :align: center
+   :alt: The Delivery Lead Time field for a subcontractor, on the Purchase tab of a product page.
+
+Product manufacturing lead time
+-------------------------------
+
+Next, navigate to the product's |BoM| by clicking the :guilabel:`Bill of Materials` smart button at
+the top of the product's page. Then, select a BoM from the list.
+
+On the |BoM|, select the :guilabel:`Miscellaneous` tab. In the :guilabel:`Manuf. Lead Time` field,
+enter the same number of days that was entered in the :guilabel:`Delivery Lead Time` field of the
+|BoM|'s product.
```

Comment:
Just checking: the manufacturing lead time is the same as the product delivery lead time?

I'm slightly confused because in the Mike's Bikes example below, the delivery lead time is 5 days, and the manufacturing lead time is 3 days